napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -326,10 +307,10 @@ def setup_ui(self, enabled=True):
         self.version_choice_text = QLabel('Version:')
         self.source_choice_dropdown = QComboBox()
 
-        if len(self._versions_pypi) is not None:
+        if len(self._versions_pypi):
             self.source_choice_dropdown.addItem(PYPI)
 
-        if len(self._versions_conda) is not None:
+        if _IS_NAPARI_CONDA_INSTALLED and len(self._versions_conda):
```

Comment:
Hm Reading this logic. 

call `len(None)` ends with TypeError. But the empty list evals to `False` and `None` evals to false so it should be changed to `if self._versions_pypi` and `if _IS_NAPARI_CONDA_INSTALLED and self._versions_conda`

Reply:
Fixed!